Motion Amendments 2019 - Advisory Council (#29)
thewca/wca-documents: documents/motions/10.2019.1 - Advisory Council.md

# SUBMISSION OF PROPOSED MOTION

**Motion number:** 10.2019.1  
**Subject:** WCA Advisory Council  
**Intent:** Rights and duties of the WCA Advisory Council  
**Submitted by:** Board of Directors  
**Date:** January 1, 2019  

# Motion

> The WCA Advisory Council is an Advisory Committee of the WCA.

Reviewer: ```suggestion
The WCA Advisory Council is an Advisory Council of the WCA.
```
This line is hilarious :D But I think it's correctly written like this.

> 1. The WCA Advisory Council is responsible for building and maintaining bilateral communication between the WCA Community and the WCA as an organization.

Reviewer: ```suggestion
1. The WCA Advisory Council is responsible for building and maintaining bilateral communication between the WCA Community and the WCA.
```

2. The WCA Advisory Council has the following rights and duties:

>    1. Establishing communication channels between the WCA Community and the WCA Board, providing direct feedback from Registered Speedcubers about the WCA as an organization.

Reviewer: ```suggestion
   1. Establishing communication channels between the WCA Community and the WCA Board, providing direct feedback from Registered Speedcubers about the WCA.
```

>       1. The WCA Advisory Council shall ensure that the demands and requests of the WCA Community are heard within the WCA organization and are addressed by the WCA Board or the appropriate Committee(s)/Team(s).

Reviewer: ```suggestion
      1. The WCA Advisory Council shall ensure that the demands and requests of the WCA Community are heard within the WCA and are addressed by the WCA Board or the appropriate Committee(s)/Team(s).
```

>    2. Recruiting Registered Speedcubers who are abel and willing to help improve the WCA organization.

Reviewer: ```suggestion
   2. Recruiting Registered Speedcubers who are able and willing to help improve the WCA.
```

>       1. The WCA Advisory Council shall provide a platform for Registered Speedcubers who are volunteering to help the WCA in any way they could.

Reviewer: Shouldn't we provide such platform? i.e. WCA Discourse.

>    3. Improving and further promoting transparency in the WCA.

Reviewer: ```suggestion
   3. Improving and further promoting transparency within the WCA.
```

>    4. Improving the communication within the WCA organization.

Reviewer: I would delete this point, this is in the scope of the WCT, no?

> 3. The WCA Advisory Council shall have the following types of representatives:

Reviewer: We haven't said anywhere what a representative is, we should introduce that before speaking of the types.
